client/components/common/FileUpload.vue

```diff
@@ -1,46 +1,34 @@
 <template>
-  <div class="file-upload">
```

Comment:
Please match new outlined style of other components
![image](https://user-images.githubusercontent.com/6253820/70279422-c3854800-17b6-11ea-9850-275b30c964aa.png)